### ISSUE TITLE
Installation Failed. "The operation couldn't be completed. Could not find matching provisioning profile"

### ISSUE THREAD

**Issue by Pacecca12** [rileytestut/AltStore#377]
**Describe the bug**
I get this error every time I try to use altserver to install altstore. I am using the Big Sur Workaround Beta and am currently on iOS 14. This bug occurs even after I have completed resetted my iPhone and even when I am using a new apple id

**To Reproduce**
1. Install the Big Sur beta for AltServer.
2. Plug iPhone into Laptop.
3. Click "Install AltStore" and pick device.
4. Enter Apple ID.
5. Error Occurs.

**Expected behavior**
I expected it to tell me that AltStore had been installed successfully.

**Desktop (please complete the following information if applicable):**
 - OS: Mac
 - Version: Big Sur

**iPhone (please complete the following information):**
 - Device: iPhone XR
 - iOS: 14.0

**Comment by 5-c-o** [comment]
Re-download/update AltStore to fix this.

**Comment by Pacecca12** [comment]
ok, so I reinstalled it. with my original apple id, I clicked on the installation for my phone, then it came up with "AltStore already installed on another device.". I clicked continue but then got another error. "The operation couldn’t be completed. (NSURLErrorDomain error -1011.)", I retried the installation and then got the "no matching provisioning file" error.

**Comment by MGSDS** [comment]
I have the same problem with my iphone 5s ios 12.4.8

**Comment by olmec73** [comment]
Having problems here to. Could be altstore 1.4 update problem?

**Comment by CrlsPerez** [comment]
I was having the problem.
But I search for updates, then update AltServer.
After this I could reinstall on my phone.

**Comment by Pacecca12** [comment]
SOLVED: I switched to the windows version and it worked. I assume there was a problem with my mac rather than altserver itself.

**Comment by adi6409** [comment]
the download link to the altserver beta Is outdated. it will download beta 1. check for updates in the altserver app then update.

**Comment by medkannouch** [comment]
Solved the problem after following @CrlsPerez solution 
Thank you @CrlsPerez

**Comment by Samko4** [comment]
Try to trust the phone with your Mac

**Comment by EVANDR4444** [comment]
I fixed this issue by installing an updated version of AltServer on my Mac (and dragging the .app into the Applications folder), then uninstalling AltStore on my phone and reinstalling using AltServer.

**Comment by Woverlin** [comment]
I see that error and I cant install newest version (1.4.1) with mac os Big Sur. How I can install AltStore in my phone ?

**Comment by TylerDKnight** [comment]
I could not solve it with the solutions mentioned so far. 

I had the error above. 
On my computer: I updated AltStore & restarted.
On my device: I deleted AltStore and its certification, then reinstalled & re-trusted it. 
Thankfully, updating to AltStore 1.4 made the above error go away and I was able to complete this installation process.
However, despite all this, the AltStore app still crashes when opened.

Any idea what to do? I followed the installation process so it seems like a bug.

Windows 10, latest AltServer (1.4), iPhone XR, iOS 13.3.1
